web/src/main/resources/messages.properties

```diff
@@ -1,6 +1,7 @@
 typeMismatch.certificate.certificationDate=Please enter a valid date like (2011-11-11).
-typeMismatch.user.discount=This field must be between 0 and 100 including.
-typeMismatch.payment.amount=This field must be like 10,0 or 10.
-typeMismatch.course.start=Please enter a valid date like (2011-11-11T00:00:00).
-typeMismatch.course.price=The price must be up to 999999999.99.
+typeMismatch.payment.amount=This field must be like 10,0 or 10 
+typeMismatch.user.discount=This field must be between 0 and 100 including
+typeMismatch.course.start=Please enter a valid date (2001-01-01)
```

Comment:
what is month and what is day?

Reply:
you can write something like 2001-20-05 for clearness

Reply:
I agree